neonvm/tools/vm-builder/main.go

```diff
@@ -231,6 +231,13 @@ else
     /neonvm/bin/echo -n {{range .Cmd}}' '{{.}}{{end}} >> /neonvm/bin/vmstarter.sh
 fi
 
+/neonvm/bin/cat <<'EOF' >>/neonvm/bin/vmstarter.sh
+
+if [ -e "/neonvm/vmstart_command_finished.fifo" ]; then
+	echo ' ' > /neonvm/vmstart_command_finished.fifo
+fi
+EOF
```

Comment:
nb: `vmstarter.sh` is fully generated by vm-builder, and run as the image's USER (for us, postgres). May be worthwhile to move this into `vmstart` instead? (both because it's running as root, and it's static)

Reply:
Good idea